### PR TITLE
#132: batch explorer should take startIndex into account when generat…

### PR DESCRIPTION
**Pull request by rombautt**
Fix for sending batched data to testpad (tested with 1.000.000 entries)

### CODE REVIEW

**Comment by rombautt** [comment]
@jgheerardyn lemme know when you created a bucket on S3, so I can upload the large test zip file. Also tell me in which folder I should put it.

File: testpad-core/src/main/java/io/yields/testpad/rest/api/ExplorerResultHandler.java [hunk 1]
```diff
@@ -89,23 +92,23 @@ public void send(String application, String timestamp, String group, String test
             int start = index * BATCH_SIZE;
```
Comment (jgheerardyn) [inline testpad-core/src/main/java/io/yields/testpad/rest/api/ExplorerResultHandler.java]: maybe better call this offset?

**Comment by jgheerardyn** [comment]
@rombautt There is now a yields-data bucket on S3. Please put the data in folder /examples/og.

Could you also verify that the access permissions to the bucket are correctly set up, i.e. that only our own IAM users have access?

**Comment by jgheerardyn** [comment]
reviewed.

File: testpad-core/src/main/java/io/yields/testpad/rest/api/ExplorerResultHandler.java (hunk 1 above)
Comment (rombautt) [inline testpad-core/src/main/java/io/yields/testpad/rest/api/ExplorerResultHandler.java]: agree, fixed

**Comment by rombautt** [comment]
@jgheerardyn had to create 2 groups for S3 access.

We are member of the S3-rw (full access) group, Adam is member of the S3-r (read-only) group.

Both files have been uploaded, WIKI has been adapted.

**Pull request merged**